zbus/src/error.rs

```diff
@@ -61,6 +62,8 @@ pub enum Error {
     InvalidSerial,
     /// The given interface already exists at the given path.
     InterfaceExists(InterfaceName<'static>, ObjectPath<'static>),
+    /// Invalid D-Bus server address.
+    DBusAddress(AddressError),
```

Comment:
We already discussed this. All the std rust naming convention is "Addr": [IpAddr](https://doc.rust-lang.org/std/net/enum.IpAddr.html), [Ipv4Addr](https://doc.rust-lang.org/std/net/struct.Ipv4Addr.html), [Ipv6Addr](https://doc.rust-lang.org/std/net/struct.Ipv6Addr.html), [SocketAddr](https://doc.rust-lang.org/std/net/enum.SocketAddr.html), [SocketAddrV4](https://doc.rust-lang.org/std/net/struct.SocketAddrV4.html), and [SocketAddrV6](https://doc.rust-lang.org/std/net/struct.SocketAddrV6.html). Please don't force me to do something I disagree with. You are free to rename after the merge. I will never agree with that.

Reply:
* I'm talking about the name of the enum variant here.
* In case of std API, we don't have any context. This is zbus, and the context of D-Bus is quite clear. There is no need for a prefix to provide context. If there is no prefix, there's no need to shorten `Address`.

> Please don't force me to do something I disagree with. You are free to rename after the merge. I can't agree with your taste, I follow patterns.

That's now how reviews work. With this statment, you're telling me that you don't intend to help maintain this code much and it will be my responsibility to fix all the bugs and answer all questions that come in (as always).

Reply:
about naming of enum: yes, I didn't want to repeat the argument later.

I value consistency more than you, it seems. 

Address is still unclear for me, even if it comes from a zbus namespace. 

I intend to maintain code that I write. But when I strongly disagree with you on taste (it's only taste here I think, and bikeshedding), I can't be forced. You are the main maintainer, I can't force you either.

Reply:
> I can't be forced. You are the main maintainer, I can't force you either.

Well one of us needs to be forced. :) I won't sign off on something I disagree with and as the maintainer, I always get the final say. If I change the names after the merge and you actually do help with maintainance this time around (I have serious doubts based on last many years of experience but I want to believe you), you'll have to deal with the names I choose anyway. What's to be gained by you not already changing the names before the merge?

Reply:
Ok.., at least you can recognize I am not as stubborn as you and I made my point clear: you do not follow rust naming convention, you have your own. I strongly disagree with this, I'll keep repeating why you are wrong, but you have the last word.

https://doc.rust-lang.org/std/index.html?search=addr :

Nothing is named "address" (for the same reason vec is not "vector" for instance, there are a lot of common programming short name, addr is one of them).

And for clarilty reasons, when such a common term as "addr" is used in a common namespace, by default, you shouldn't pollute the namespace and use a more descriptive name. Hence `DBusAddr` is the right Rust naming choice.

Reply:
> you do not follow rust naming convention

Huh? I explained my reasons for the naming. This is **not** any established Rust convention. Just because std uses some naming convention, it doesn't necessarily make it a convention. We already use `Address` in zbus and I didn't hear any compelling reason to change it.

Anyway, naming isn't something you're likely to convince me on. That's something you must change for this PR to be accepted. Plus I'm becoming sure as I think and discuss more about it that I don't want a split but a module.